x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ElasticServiceAccounts.java

```diff
@@ -150,13 +150,34 @@ final class ElasticServiceAccounts {
             null
         )
     );
+    private static final ServiceAccount FLEET_REMOTE_ACCOUNT = new ElasticServiceAccount(
+        "fleet-server-remote",
+        new RoleDescriptor(
+            NAMESPACE + "/fleet-server-remote",
+            new String[] { "manage_own_api_key" },
```

Comment:
Do we need to add the `monitor` privileges? I think we currently add this to the api key we generate for agent

Reply:
done